src/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -29,6 +29,7 @@ IN_PROCESS_APPLICATION::IN_PROCESS_APPLICATION(
 
     InitializeSRWLock(&m_srwLock);
     m_pConfig = pConfig;
+    html500Page = GetStaticHtml500Content();
```

Comment:
Why store the error HTML in a member, rather than calling `GetStaticHtml500Content()` every time the error HTML is needed?  If it's for perf, this seems unnecessary since the HTML is only used in error cases.

Reply:
I still wouldn't bother caching this but I agree there's not a big difference either way.